README.md

```diff
@@ -224,7 +224,7 @@ Specify 7. It is possible to copy the Specify 6 install from another
 Linux system to avoid the need to install Java on the server.
 
 ```shell
-wget https://update.specifysoftware.org/6802/Specify_unix_64.sh
+wget https://update.specifysoftware.org/Specify_unix_64.sh
```

Comment:
@CarolineDenis you selected a wrong merge target for this PR
whatever branch you created `issue-4076` from, that is the branch it should be merging in